server/app/api/controllers/users.js

```diff
@@ -160,6 +161,26 @@ module.exports = {
       });
   },
 
+  changePassword: function (req, res, next) {
+    const {id} = req.params
+    const {password} = req.body;
```

Comment:
@surekhag it would be better to have some validation for password, otherwise user can misuse API or there could be security issue. Always validate your data when you update/delete anything.